authentication/views.py

```diff
@@ -1,13 +1,43 @@
 from django.shortcuts import render
 from django.contrib.auth import login,logout,authenticate
+from django.views.decorators.csrf import csrf_exempt
 # Create your views here.
-
-
+from django.shortcuts import redirect,render
+from django.contrib.auth.models import User
+from .forms import SignUpForm
+from django.http import HttpResponse
+# @csrf_exempt
 def loginView(request):
-    pass
-
+    if request.user.is_authenticated:
+        return redirect('index')
+    if request.method == "POST":
+        username = request.POST['Username']
+        password =  request.POST['Password']
```

Comment:
You are directly accessing POST data without checking if it even exists. This may lead to server crash if a user access this endpoint with invalid request data. The good behavior would have been to throw a client error (400), rather than server error (500).